release/scan-image-vulnerabilities/README.md

```diff
@@ -31,49 +31,40 @@ Image name without the registry prefix. The action will automatically prepend `q
 
 Example: `"rhacs-eng/main"`
 
-Required: Yes
-
 #### version
 
 Image version tag to scan.
 
 Example: `"3.76.1"`
 
-Required: Yes
-
 #### wait-limit
 
 Maximum time in seconds to wait for the image to be available on Quay.io before failing.
 
-Required: No
-
 Default: `"7200"` (2 hours)
 
 #### summary-prefix
 
-Title prefix for the vulnerability report in the GitHub step summary. This helps identify which image the scan results correspond to when multiple scans are performed in a workflow.
-
-Example: `"Image Scan Results"`
-
-Required: Yes
+Prefix for the vulnerability report in the GitHub step summary. Use this to help users of the action classify images into groups when multiple matrix scans are performed in a workflow.
+	
+Example: `"Upstream Image Scan Results"`
 
 #### quay-bearer-token
 
 Bearer token for authenticating with the Quay.io API. This is required by the wait-for-image action to check if the image is available.
 
-Required: Yes
-
 #### central-url
 
 URL of the ACS/RHACS Central instance to use for scanning.
 
 Example: `"https://central.example.com"`
 
-Required: Yes
-
 ## Usage
 
-The action requires credentials for ACS Central to be available. It integrates with the `stackrox/central-login@v1` action which uses OIDC authentication.
+The action integrates with the [stackrox/central-login](https://github.com/stackrox/central-login) action, which uses OIDC login for authentication of the `roxctl` CLI.
+The ACS Central needs to be configured to allow exchanging tokens from GitHub Actions workflow runs. 
+
+Additionally, an image integration for Quay.io needs to be configured in the ACS Central.
```

Comment:
Suggestion:
```suggestion
Additionally, an image integration for Quay.io must be configured in the ACS Central so that it can pull images requested for scanning.
```